packages/medusa/src/loaders/module.ts

```diff
@@ -0,0 +1,60 @@
+import { asFunction, asValue } from "awilix"
+import { trackInstallation } from "medusa-telemetry"
+import { ConfigModule, Logger, MedusaContainer } from "../types/global"
+import { ModulesHelper } from "../utils/module-helper"
+
+type Options = {
+  container: MedusaContainer
+  configModule: ConfigModule
+  logger: Logger
+}
+
+const ModuleHelper = new ModulesHelper()
+
+export default async ({
+  container,
+  configModule,
+  logger,
+}: Options): Promise<void> => {
+  const moduleResolutions = configModule?.moduleResolutions ?? {}
+
+  for (const resolution of Object.values(moduleResolutions)) {
+    try {
+      const loadedModule = await import(resolution.resolutionPath!)
+
+      const moduleLoaders = loadedModule.loaders
+      if (moduleLoaders) {
+        for (const loader of moduleLoaders) {
```

Comment:
suggestion:
```ts
const modulesLoaders = loadedModule.loaders ?? []
for (const loader of moduleLoader) ...
```

Reply:
It can be the same for the services below and improve readability by reducing indentation and having the same output